code/modules/projectiles/guns/ego_gun/teth.dm

```diff
@@ -80,4 +80,3 @@
 	weapon_weight = WEAPON_HEAVY
```

Comment:
Standalone and unimplemented code, just needs to be removed.

Reply:
That's not my code. that's dream's code. Plus I'm pretty sure that means that the weapon is two handed.